Examples/SyncUps/SyncUps/AppFeature.swift

```diff
@@ -71,18 +71,13 @@ struct AppView: View {
 }
 
 #Preview {
-  AppView(
-    store: Store(
-      initialState: AppFeature.State(
-        syncUpsList: SyncUpsList.State(
-          syncUps: [
-            .mock,
-            .productMock,
-            .engineeringMock,
-          ]
-        )
-      )
-    ) {
+  @Shared(.syncUps) var syncUps: IdentifiedArrayOf<SyncUp> = [
```

Comment:
I had some sync-ups cleanup on the shared branch that snuck into this. The PR review can stay focused on the `Sources` directory, I can also cherry-pick the sync-ups changes back into the beta branch if that makes it easier to review.